VideoWeb/VideoWeb/Mappings/ParticipantStatusResponseForVhoMapper.cs

```diff
@@ -23,7 +24,8 @@ public class ParticipantStatusResponseForVhoMapper : IMapTo<Conference, IEnumera
                 .Select(x =>
                 {
                     var status = Enum.Parse<ParticipantStatus>(x.ParticipantStatus.ToString());
-                    var hostInHearing = hostsInHearings.Where(j => j.Username == x.Username && j.Id != x.Id);
+                    var hostInHearing = hostsInHearings.Where(j => j.Username == x.Username && j.Id != x.Id && 
+                        (j.Status == ParticipantState.InHearing || j.Status == ParticipantState.Available));
```

Comment:
Should we check for InConsultation here as well?

Reply:
That makes sense. I will add that